src/test/data/JsonAddressBookStorageTest/invalidAndValidPersonAddressBook.json

```diff
@@ -1,11 +1,9 @@
 {
   "persons": [ {
-    "name": "Valid Person",
-    "phone": "9482424",
-    "address": "4th street"
+    "tagName": "Valid Person",
```

Comment:
This may not work. But for now it's okay.

Reply:
Amend